cypress/e2e/Labels/utilsAndConstants/labelsUtils.ts

```diff
@@ -180,28 +137,19 @@ export function checkUpdatedLabelExists(options: {
   });
 
   if (labelName) {
-    cy.dataId({ value: "stHorizontalBlock" })
+    cy.dataId("stHorizontalBlock")
       .should("exist")
       .contains(labelName)
       .parents('div[data-testid="stHorizontalBlock"]') // finds all the parents of the element with labelName
       .should("exist")
       .within(() => {
         condition &&
-          cy
-            .dataId({ value: "stVerticalBlock" })
-            .contains(condition)
-            .should("exist");
+          cy.dataId("stVerticalBlock").contains(condition).should("exist");
 
-        rank &&
-          cy
-            .dataId({ value: "stVerticalBlock" })
-            .contains(rank)
-            .should("exist");
+        rank && cy.dataId("stVerticalBlock").contains(rank).should("exist");
```

Comment:
@susannahmarcus , I don't understand what happened with formatting (and there are bunch of other cases like this).  Code in lines 195 - 199 is new since yesterday. 
Did you manually remove `{ value:` part and then because line became shorter it got collapsed into a single line? 
If you did not manually remove it, it's almost like another version `prettier` is doing it...

Reply:
formatting:
in JS, if you're looking to see if something exists, you can just have the check structured like this. It basically says `if rank is defined, do this`.
As for the `{ value: "stVerticalBlock" }` turning into `"stVerticalBlock" ` it's for the custom utils I've added. I changed their arguments structure because I made it an object before since there was a optional timeout increase, but noticed that nothing was using it and it was adding the step of typing `{value:}` for nothing.